go/vt/vtctl/workflow/server.go

```diff
@@ -1053,16 +1115,28 @@ func (s *Server) getWorkflowState(ctx context.Context, targetKeyspace, workflowN
 	return ts, state, nil
 }
 
-func (s *Server) getWorkflowCopyStates(ctx context.Context, tablet *topo.TabletInfo, id int64) ([]*vtctldatapb.Workflow_Stream_CopyState, error) {
+func (s *Server) getWorkflowCopyStates(ctx context.Context, tablet *topo.TabletInfo, streamIds []int64) ([]*vtctldatapb.Workflow_Stream_CopyState, error) {
 	span, ctx := trace.NewSpan(ctx, "workflow.Server.getWorkflowCopyStates")
 	defer span.Finish()
 
 	span.Annotate("keyspace", tablet.Keyspace)
 	span.Annotate("shard", tablet.Shard)
 	span.Annotate("tablet_alias", tablet.AliasString())
-	span.Annotate("vrepl_id", id)
+	var strStreamIds []string
+	for _, streamId := range streamIds {
+		strStreamIds = append(strStreamIds, strconv.FormatInt(streamId, 10))
+	}
+	span.Annotate("stream_ids", strings.Join(strStreamIds, ","))
```

Comment:
IMO it would adequate in this case to do this since there can't be any whitespace within the values:
```
span.Annotate("stream_ids", fmt.Sprintf("%v", strStreamIds))
```

Or this (https://go.dev/play/p/VdgdivLDiA8):
```
span.Annotate("stream_ids", fmt.Sprintf("%#v", strStreamIds))
```